scripts/common/rook.sh

```diff
@@ -67,13 +67,9 @@ function remove_rook_ceph() {
     # make sure there aren't any PVs using rook before deleting it
     all_pv_drivers="$(kubectl get pv -o=jsonpath='{.items[*].spec.csi.driver}')"
     if echo "$all_pv_drivers" | grep "rook" &>/dev/null ; then
-        # do stuff
-        printf "%b" "$RED"
-        printf "ERROR: \n"
-        printf "There are still PVs using rook-ceph.\n"
-        printf "Remove these PVs before continuing.\n"
-        printf "%b" "$NC"
-        exit 1
+        logFail "There are still PVs using rook-ceph."
+        logFail "Remove these PV(s) before continuing."
```

Comment:
@laverya I leaved it as before. 
Honest, I do not understand it. 
Why we say to remove instead of remove? 

We can check it an follow up

Reply:
we don't want to do the removal ourselves because that would delete data
the PVs should have all been removed by pvmigrate